src/app/dashboard/components/Dashboard.jsx

```diff
@@ -0,0 +1,12 @@
+'use client'
+
+import SidebarDashboard from './sidebar/Sidebar'
+
+function Dashboard () {
+  return (
+    <div className='flex h-full'>
+      <SidebarDashboard />
```

Comment:
One problem with this approach is that the `<url>/dashboard` page exists and is accessible, despite not being useful:
<img width="1914" alt="image" src="https://github.com/user-attachments/assets/59f79ad3-0303-420b-877c-bcadfb48a00d">

Possible ways to avoid this:
- just not define this page, only the `dashboard/overview` and `dashboard/contracts` one. That's probably the easiest.
- automatically route towards `dashboard/overview` when the `dashboard` page is reached.